blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubServerContainer.java

```diff
@@ -9,7 +9,8 @@
 import io.jenkins.blueocean.commons.ServiceException;
```

Comment:
We should switch all this over to use the URL as the index. Perhaps on a follow up PR?